From 8091f1f8f716fb33ecfd67f9c2adb54e3a4ca883 Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Thu, 18 Oct 2018 15:24:59 +0300
Subject: [PATCH 117/211] media: platform: soc_camera: rcar_vin: Update R-Car
 V3M bindings

This updates V3M compatible string to "renesas,vin-r8a77970".
While at it use more descriptive name for the V3M ifmd array.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 drivers/media/platform/soc_camera/rcar_vin.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/media/platform/soc_camera/rcar_vin.c b/drivers/media/platform/soc_camera/rcar_vin.c
index 291d115..ff4e521 100644
--- a/drivers/media/platform/soc_camera/rcar_vin.c
+++ b/drivers/media/platform/soc_camera/rcar_vin.c
@@ -439,7 +439,7 @@ static const struct vin_gen3_ifmd vin_m3n_vc_ifmd[] = {
 	},
 };
 
-static const struct vin_gen3_ifmd vin_v3_vc_ifmd[] = {
+static const struct vin_gen3_ifmd vin_v3m_vc_ifmd[] = {
 	{ 0x0000,
 		{
 			{RCAR_CSI40, RCAR_VIRTUAL_CH0},
@@ -2770,7 +2770,7 @@ static struct soc_camera_host_ops rcar_vin_host_ops = {
 
 #ifdef CONFIG_OF
 static const struct of_device_id rcar_vin_of_table[] = {
-	{ .compatible = "renesas,vin-r8a7797", .data = (void *)RCAR_V3M },
+	{ .compatible = "renesas,vin-r8a77970", .data = (void *)RCAR_V3M },
 	{ .compatible = "renesas,vin-r8a77965", .data = (void *)RCAR_M3N },
 	{ .compatible = "renesas,vin-r8a7796", .data = (void *)RCAR_M3 },
 	{ .compatible = "renesas,vin-r8a7795", .data = (void *)RCAR_H3 },
@@ -3188,7 +3188,7 @@ static int rcar_vin_probe(struct platform_device *pdev)
 			ifmd = VNCSI_IFMD_DES1;
 			break;
 		case RCAR_V3M:
-			gen3_ifmd_table = vin_v3_vc_ifmd;
+			gen3_ifmd_table = vin_v3m_vc_ifmd;
 			break;
 		default:
 			BUG();
@@ -3358,7 +3358,7 @@ static int rcar_vin_resume(struct device *dev)
 		ifmd = VNCSI_IFMD_DES1;
 		break;
 	case RCAR_V3M:
-		gen3_ifmd_table = vin_v3_vc_ifmd;
+		gen3_ifmd_table = vin_v3m_vc_ifmd;
 		break;
 	default:
 		return 0;
-- 
2.7.4

